### ISSUE TITLE
CRAN logs website down

### ISSUE THREAD

**Issue by Jeff-Thompson12** [pharmaR/riskassessment#424]
To populate the community usage metrics in the application, we are running the function `cranlogs::cran_downloads()`. It pulls the download information from [here](http://cranlogs.r-pkg.org/downloads/daily/). I personally have experienced this webpage being down several times in the past week. This results in no data being pulled in.

I think it would be nice to alert the user (or completely disable package upload) if needed websites are returning 404 errors.

**Comment by AARON-CLARK** [comment]
@Jeff-Thompson12, I had this happen to me as well. Pretty frustrating because the progress bar took forever.

Forever, we've actually been kind of naughty by deviating from `riskmetric` and [summing up the number of downloads](https://github.com/pharmaR/riskassessment/blob/d31168373ae98c321f1f3e100c317863bf120cbc/R/utils.R#L328-L336) in the last 12 months ourselves for the Community Usage tab, ignoring the what `riskmetric` inserts into the DB. We do this so that our month-by-month plot doesn't include partial months, and we wanted the sum of the last 12 months displayed on the plot so our numbers checked out with what's displayed in the card. So, if we can't pull from `cran_downloads` due to a 404 error, maybe we don't stop the upload to db and just display the `riskmetric` assessment value, which should exist in the DB (as long as `riskmetric` isn't also getting denied with a 404 error from whatever source they are using). Just note, the risk score is always based on `riskmetric`s assessment and not ours.

FYI, as a result of inspecting this, I also just realized that there has been a major bug in what we report on the card for the number of downloads in the last 12 months. Instead of summarizing the last 12 months, we just summarized all months from the last year. So if you run the code today, Feb 2023, it would just sum all of 2022, ignoring Jan 2023. Whoops! The problem would get worse the farther you are into the year. For example, if you ran an assessment in Dec 2022, it would only sum months from 2021! Yikes. PR #437 opened to fix. Note this doesn't fix this issue though.
